File: leet_code/string/168/168.dart
class Solution {
  String convertToTitle(int columnNumber) {
    String result = '';
    while (columnNumber > 0) {
      columnNumber--; 
      int remainder = columnNumber % 26;
      result = String.fromCharCode(remainder + 65) + result;
      columnNumber = columnNumber ~/ 26;
    }
    return result;
  }
}

File: leet_code/string/168/168.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        result = []
        while columnNumber > 0:
            columnNumber -= 1  
            remainder = columnNumber % 26
            result.append(chr(remainder + ord('A')))
            columnNumber //= 26
        return ''.join(result[::-1])
